app/views/charge.njk

```diff
@@ -27,7 +47,7 @@
     </div>
 
   {% if allowApplePay or allowGooglePay %}
-    <form id="payment-request-container" class="hidden">
+    <form id="web-payments-container">
```

Comment:
Changing the ID in the HTML and the CSS in the external stylesheet is risky (because one can end up with the new HTML but the old CSS or vice-versa) but since no live services have Apple Pay or Google Pay enabled yet, I think this should be OK